connectivity/FEATURE_BLE/source/cordio/driver/CordioHCITransportDriver.cpp

```diff
@@ -32,6 +37,9 @@ void CordioHCITransportDriver::on_data_received(uint8_t* data, uint16_t len)
 {
     while (len) {
         uint8_t chunk_length = std::min(len, (uint16_t) std::numeric_limits<uint8_t>::max());
+#if MBED_CONF_CORDIO_TRACE_HCI_PACKETS
+        tr_debug("LL->HOST: %s", trace_array(data, chunk_length));
```

Comment:
It won't work well on all platforms, depending on the speed of the HCI link we may print one character at the time instead of the complete packet. 

Ideally that trace would be in `hciCoreRecv` the WSF trace has the `HCI_PDUMP_*` macros for this.

Reply:
It is now using HCI_PDUMP